Longest_Palindromic_Subsequence/Longest_Palindromic_Subsequence.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+    LONGEST PALINDROMIC SUBSEQUENCE
+
+    The task is to find the length of the longest possible subsequence of a given word which is      a palindrome.
+    A palindrome is a string that is same when read from any side.
+
+*/
+#include<bits/stdc++.h>
+using namespace std;
+#define lint long long int
+
+/*
+    We use Dynamic Programming approach to solve this problem.
+    A 2D matrix M of size n*n is created where n is the length of the given string.
+    Here the row i denotes the char S[i] of the string while column j denotes char R[j], where
+    R is the reverse of the string S.
+    Thus, M[i][j] denotes the max length of longest common subsequence between the string 
+    S[0...i] and R[0..j]. Hence M[n][n] will compute the longest common subsequence of S[0..n]       and R[0..n], which is the required answer.
+*/
+void lps(string str) {
+  string rev = str;
```

Comment:
4 space indentation not followed. Please take a look at previous PR's to get a better idea as to how you need to indent the code